src/main/scala/cromwell/util/Backoff.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+sealed trait Backoff {
+  def backoffMillis: Long
+  def next: Backoff
+}
+
+final class InitialGapBackoff(initialGapMillis: FiniteDuration, googleBackoff: ExponentialBackOff) extends Backoff {
```

Comment:
I wasn't able to define a second constructor with a case class - IntelliJ was whining.